deployment/mergebranch-multiarch-agent-deployment/ServiceGroupRoot/Scripts/pushAgentToAcr.sh

```diff
@@ -66,17 +64,27 @@ fi
 echo "Login cli using managed identity"
 az login --identity
 if [ $? -eq 0 ]; then
-  echo "Logged in successfully"
+  echo "az logged in successfully"
 else
   echo "-e error failed to login to az with managed identity credentials"
   exit 1
-fi     
+fi
 
-echo "Pushing ${AGENT_IMAGE_FULL_PATH} to ${ACR_NAME} with force option set to ${AZ_ACR_IMPORT_FORCE}"
-az acr import --name $ACR_NAME --source $SOURCE_IMAGE_FULL_PATH --image $AGENT_IMAGE_FULL_PATH $AZ_ACR_IMPORT_FORCE
+az acr login -n containerinsights
```

Comment:
this is same acr name coming as the value of ${ACR_NAME}, right? If yes, let use the ${ACR_NAME} and this way in future if we can change the ACR_NAME pipeline variable value and that should work.

Reply:
changed